python/lsst/ts/standardscripts/maintel/calibration/power_on_tunablelaser.py

```diff
@@ -0,0 +1,178 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["PowerOnTunableLaser"]
+
+import asyncio
+
+import yaml
+from lsst.ts import salobj
+from lsst.ts.idl.enums import TunableLaser
+
+
+class PowerOnTunableLaser(salobj.BaseScript):
+    """Starts propagating the Tunable Laser for functional
+    testing.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    """
+
+    def __init__(self, index, add_remotes: bool = True):
```

Comment:
Please, drop the `add_remotes` here... We no longer use this, and it doesn't seems like you are using it.